docs/reference/search/filter-search-results.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[filter-search-results]]
+== Filter search results
+
+You can use two methods to filter search results:
```

Comment:
I'd put the emphasis more on what the user is trying to accomplish than the mechanics at this level. Something like:

You can filter search results by including a filter clause within the query itself, or by applying a post filter. When you include a filter clause in a Boolean query, the filter applies to both the search hits and aggregations. When you use a post filter, the filter only applies to the search hits. Using a post filter enables you calculate aggregations based on a broader result set, and then further narrow the results.